### DISCUSSION TITLE
Adding M1 Mac Breeze Troubleshooting Details

### DISCUSSION THREAD

**Discussion by nijanthan-dev** [apache/airflow#17360] [Q&A] [answered]
Hi @potiuk,

I want to add troubleshooting information while installing/setting up Breeze on a M1 Mac. Should I add those details to this [troubleshooting section within Breeze](https://github.com/apache/airflow/blob/main/BREEZE.rst#troubleshooting) or create a new TROUBLESHOOTING.rst file?

**Comment by potiuk** [comment, marked as answer]
Add it to the troubleshooting section please.
